src/styles.py

```diff
@@ -199,12 +208,11 @@ def _on_style_change(self, override: bool = False) -> None:
         headerbar.provider = Gtk.CssProvider()
 
         # Set headerbar color and contrast
-        bg_color = self._selected_style_params["figure.facecolor"]
-        contrast = utilities.get_luminance(bg_color)
-        color = "@dark_5" if contrast > 150 else "@light_1"
+        color = "@light_1" if _is_style_bright(self._selected_style_params) \
```

Comment:
```suggestion
        bg_color = self._selected_style_params["figure.facecolor"]
        color = "@light_1" if _is_style_bright(self._selected_style_params) \
```

Sloppy review from my part, the `bg_color` declaration was missing, that should be there